installers/windows/ReadMe.htm

```diff
@@ -1,13 +1,13 @@
 <html>
 <head>
-<title>Monero Helium Hydra GUI Wallet</title>
+<title>Monero Lithium Luna GUI Wallet</title>
 </head>
 
 <body style="font-family: Arial, Helvetica, sans-serif">
-<h1>Monero Helium Hydra GUI Wallet</h1>
+<h1>Monero Lithium Luna GUI Wallet</h1>
 
   <p>Copyright (c) 2014-2018, The Monero Project<br>
-  Date: September 19, 2017</p>
+  Date: March 18, 2018</p>

```

Comment:
What is that for ? If it's a copyright date, then it's wrong in the first place.

Reply:
That was my idea of putting something like a "Written on this and this date" into the file, as an information for the reader. Motivated by things like articles I sometimes find on the Internet with no such date and thus impossible to judge.

If that's no good idea I zap it, no problem.

Reply:
Seems ok to me.